Apps/APN-UIKit/APN UIKit/APN UIKit/View/LoginViewController.swift

```diff
@@ -0,0 +1,57 @@
+import UIKit
+
+class LoginViewController: UIViewController {
+
+    static func newInstance() -> LoginViewController {
+        UIStoryboard.getViewController(identifier: "LoginViewController")
+    }
+    // MARK: - Outlets
+    @IBOutlet weak var emailTextField: ThemeTextField!
+    @IBOutlet weak var firstNameTextField: ThemeTextField!
+
+    var loginRouter: LoginRouting?
+    
+    override func viewWillAppear(_ animated: Bool) {
+         super.viewWillAppear(animated)
+        self.navigationController?.isNavigationBarHidden = false
```

Comment:
Another place where this code is used and an extension function `self.hideNavigationBar()` can be used.